### PR TITLE
feat!: Update raster CSW record to new design (MAPCO-4265, MAPCO-4232)

### PR DESCRIPTION
**Pull request by RonitKissis**

### CODE REVIEW

File: src/yaml/updateLayerMetadata.yaml [hunk 1]
```diff
@@ -59,19 +59,19 @@ components:
...
           type: string
           description: Ingestion date
           format: date-time
-        sourceDateStart:
+        imagingTimeBeginUTC:
           type: string
           description: Oldest imaging date
           format: date-time
-        sourceDateEnd:
+        imagingTimeEndUTC:
```
Comment (asafMasa) [inline src/yaml/updateLayerMetadata.yaml:74]: remove "layerPolygonParts", raw etc.

File: src/yaml/updateLayerMetadata.yaml [hunk 2]
```diff
@@ -59,19 +59,19 @@ components:
         producerName:
           type: string
           description: creator name default IDFMU
-        creationDate:
+        creationDateUTC:
```
Comment (asafMasa) [inline src/yaml/updateLayerMetadata.yaml:62]: srsId change to srs

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]
Great Job!

**Review by asafMasa — APPROVED** [review]

File: src/yaml/fullLayerMetadata.yaml [hunk 3]
```diff
@@ -32,3 +32,22 @@ components:
               - image/png
               - image/jpeg
               description: Image format associated with content type (mime)
+            minResolutionDeg:
+              type: number
+              minimum: 1.67638e-7
```
Comment (alebinson) [inline src/yaml/fullLayerMetadata.yaml:37]: R U sure about this number?

File: src/yaml/fullLayerMetadata.yaml [hunk 4]
```diff
@@ -32,3 +32,22 @@ components:
...
+              maximum: 0.703125
+              format: double
+              description:  min resolution of layer in degrees/pixel
+            minResolutionMeter:
+              type: number
+              minimum: 0.0185
```
Comment (alebinson) [inline src/yaml/fullLayerMetadata.yaml:43]: R U sure about this number?

Comment (RonitKissis) [reply src/yaml/fullLayerMetadata.yaml:43]: 22: 0.0185  - from mc utils

File: src/yaml/fullLayerMetadata.yaml (hunk 3 above)
Comment (RonitKissis) [reply src/yaml/fullLayerMetadata.yaml]: yes, i checked but updated to the exact value from mc-utils

**Review by netanelC — APPROVED** [review]

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**